internal/controller/config/agent_config.go

```diff
@@ -167,61 +169,24 @@ func (a *AgentConfig) ApplyToAgentStart(ctr *corev1.Container) {
 	appendToEnvOpt(ctr, "BUILDKITE_AGENT_JOB_VERIFICATION_NO_SIGNATURE_BEHAVIOR", a.VerificationFailureBehavior)
 }
 
-// applyToBootstrap adds env vars assuming ctr is a checkout or command container.
-func (a *AgentConfig) applyToBootstrap(ctr *corev1.Container) {
-	a.applyCommonTo(ctr)
-	// Note that these "buildkite-agent start"-like options are applied to
-	// containers running "buildkite-agent bootstrap". So e.g. noPTY:true must
-	// be inverted to pty:false, as the agent would normally.
-	appendNegatedToEnvOpt(ctr, "BUILDKITE_PTY", a.NoPTY)
-	appendNegatedToEnvOpt(ctr, "BUILDKITE_COMMAND_EVAL", a.NoCommandEval)
-	appendNegatedToEnvOpt(ctr, "BUILDKITE_LOCAL_HOOKS_ENABLED", a.NoLocalHooks)
-	appendNegatedToEnvOpt(ctr, "BUILDKITE_PLUGINS_ENABLED", a.NoPlugins)
-	appendBoolToEnvOpt(ctr, "BUILDKITE_PLUGIN_VALIDATION", a.PluginValidation)
-}
-
-// ApplyToCheckout adds env vars assuming ctr is a checkout container.
-func (a *AgentConfig) ApplyToCheckout(ctr *corev1.Container) {
+func (a *AgentConfig) ApplyToCommand(ctr *corev1.Container) {
 	if a == nil || ctr == nil {
 		return
 	}
-	a.applyToBootstrap(ctr)
-}
-
-// ApplyToCommand adds env vars assuming ctr is a command container.
-func (a *AgentConfig) ApplyToCommand(ctr *corev1.Container) {
-	if a == nil || ctr == nil {
+	if a.SigningJWKSVolume == nil {
 		return
 	}
-	a.applyToBootstrap(ctr)
-	// Signing happens either with "pipeline upload" or "tool sign", so the
-	// signing side of any key needs to be attached to the command container,
-	// not the agent start container or checkout containers.
```

Comment:
These might be  useful comments to keep?

Reply:
Done!